src/extension/dxf/core/src/main/java/org/geoserver/wfs/response/dxf/AbstractDXFWriter.java

```diff
@@ -1,4 +1,4 @@
-/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
```

Comment:
Our latest procedures for copyrights only demand for a header to be there, but we do not care for the dates anymore (research showed they are not really legally binding). No need to change the pull request, just remember it's not a requirement anymore for the future).